Podfile

```diff
@@ -345,6 +345,17 @@ target 'WordPressStatsWidgets' do
 
   shared_with_all_pods
   shared_with_networking_pods
+  shared_with_extension_pods
+
+  wordpress_ui
+end
+
+target 'JetpackStatsWidgets' do
+  project 'WordPress/WordPress.xcodeproj'
+
+  shared_with_all_pods
+  shared_with_networking_pods
+  shared_with_extension_pods
```

Comment:
nitpick: These two targets are under the "iOS 14 Today Widget" comment header:
<img src="https://user-images.githubusercontent.com/1898325/197844710-cb89d19a-92e1-44cf-9a96-71bc438c3454.png" alt="" width="550"> 

Since these are Home Screen Widget**s**, this comment might be misleading and more so now with the addition of `JetpackStatsWidgets`.

Reply:
Sorry for the noise, I should have hit "Start a review" instead of "Add single comment" in GitHub's UI.